react-ui/src/pages/analyzeVotes/AnalyzeVotes.jsx

```diff
@@ -166,95 +166,103 @@ function AnalyzeVotes() {
                   return <em>Loading...</em>;
                 }
 
-                return selected.name;
+                return `${selected.name} (${format(parseISO(selected.date), 'MMM yy')})`;
```

Comment:
Since this is supposed to match the value within `MenuItem` perhaps a function to standardize would be better?
```js
const formattedContest = (name, date) => `${name} (${format, parseISO(date), 'MMM yy')})`;
```

Reply:
Done!